src/components/dataset/new.vue

```diff
@@ -0,0 +1,152 @@
+<!--
+Copyright 2024 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <modal id="dataset-new" :state="state" :hideable="!awaitingResponse" backdrop
+    @hide="hideOrComplete">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <template v-if="step === 0">
+        <div class="modal-introduction">
+          <p>{{ $t('introduction[0]') }}</p>
+          <i18n-t tag="p" keypath="moreInfo.helpArticle.full">
+            <template #helpArticle>
+              <doc-link to="central-entities/">{{ $t('moreInfo.helpArticle.helpArticle') }}</doc-link>
+            </template>
+          </i18n-t>
+        </div>
+        <form @submit.prevent="submit">
+          <form-group ref="name.value" v-model.trim="name"
```

Comment:
Use two refs, one for the FormGroup component and one for the name data.
```
ref="nameGroup"
v-model.trim="name"
...
const nameGroup = ref(null);
const name = ref('');
```

...

```
@shown = 'nameGroup.focus()'
```